pkg/alertmanager/multitenant.go

```diff
@@ -817,7 +817,7 @@ func createTemplateFile(dataDir, userID, fn, content string) (bool, error) {
 	}
 
 	dir := filepath.Join(dataDir, "templates", userID, filepath.Dir(fn))
-	err := os.MkdirAll(dir, 0755)
+	err := os.MkdirAll(dir, 0750)
```

Comment:
Is this creation mask intentionally different from the others? it seems to be user and group whereas others are just user?

Reply:
Not intentionally, I'll reduce this to 0700 as well.